### PR TITLE
Fix a ULT fail issue when driver dynamically links gmmlib.

### PR DESCRIPTION
**Pull request by walter-bai**
Fix a ULT fail issue when driver dynamically links gmmlib.

### CODE REVIEW

File: media_driver/linux/ult/ult_app/driver_loader.h [hunk 1]
```diff
@@ -87,7 +87,7 @@ struct DriverSymbols
     void Clear()
```
Comment (dvrogozh) [inline media_driver/linux/ult/ult_app/driver_loader.h]: Folks, I strongly suggest to remove constructor and Clear() entirely, You have class with all public members C++11 already has a way to reset a structure to default value-initialization which is: m_drvSyms = {}.

File: media_driver/linux/ult/ult_app/driver_loader.h [hunk 2]
```diff
@@ -96,7 +96,7 @@ struct DriverSymbols
     bool Initialized() const
     {
         auto p = (const void * const *)this;
-        for (auto i = 0; i < sizeof(this) / sizeof(const void *); i++)
+        for (auto i = 0; i < sizeof(*this) / sizeof(const void *); i++)
```
Comment (dvrogozh) [inline media_driver/linux/ult/ult_app/driver_loader.h]: Folks, please, remove this and check pointers directly. This is simply a freak over complex implementation of very simple thing: check 5 pointers that they are not NULL. You don't need any loop here!!

File: media_driver/linux/ult/ult_app/driver_loader.h (hunk 1 above)
Comment (walter-bai) [reply media_driver/linux/ult/ult_app/driver_loader.h]: Thank you for catching this, "m_drvSyms = {}" is much better than a Clear function.

File: media_driver/linux/ult/ult_app/driver_loader.h (hunk 2 above)
Comment (walter-bai) [reply media_driver/linux/ult/ult_app/driver_loader.h]: Yes, checking 5 pointers is simple and easy to understand than previous code.

**Comment by yangnin** [comment]
It has been combined with PR#270. close it.

**Pull request closed without merging**